tests/components/alarm_control_panel/test_device_trigger.py

```diff
@@ -207,51 +207,46 @@ def calls(hass):
     hass.states.async_set("alarm_control_panel.entity", STATE_ALARM_TRIGGERED)
     await hass.async_block_till_done()
     assert len(calls) == 1
-    assert calls[0].data[
-        "some"
-    ] == "triggered - device - {} - pending - triggered - None".format(
-        "alarm_control_panel.entity"
+    assert (
+        calls[0].data["some"]
+        == "triggered - device - alarm_control_panel.entity - pending - triggered - None"
     )
 
     # Fake that the entity is disarmed.
     hass.states.async_set("alarm_control_panel.entity", STATE_ALARM_DISARMED)
     await hass.async_block_till_done()
     assert len(calls) == 2
-    assert calls[1].data[
-        "some"
-    ] == "disarmed - device - {} - triggered - disarmed - None".format(
-        "alarm_control_panel.entity"
+    assert (
+        calls[1].data["some"]
+        == "disarmed - device - alarm_control_panel.entity - triggered - disarmed - None"
     )
 
     # Fake that the entity is armed home.
     hass.states.async_set("alarm_control_panel.entity", STATE_ALARM_PENDING)
     hass.states.async_set("alarm_control_panel.entity", STATE_ALARM_ARMED_HOME)
     await hass.async_block_till_done()
     assert len(calls) == 3
-    assert calls[2].data[
-        "some"
-    ] == "armed_home - device - {} - pending - armed_home - None".format(
-        "alarm_control_panel.entity"
+    assert (
+        calls[2].data["some"]
+        == "armed_home - device - alarm_control_panel.entity - pending - armed_home - None"
     )
 
     # Fake that the entity is armed away.
     hass.states.async_set("alarm_control_panel.entity", STATE_ALARM_PENDING)
     hass.states.async_set("alarm_control_panel.entity", STATE_ALARM_ARMED_AWAY)
     await hass.async_block_till_done()
     assert len(calls) == 4
-    assert calls[3].data[
-        "some"
-    ] == "armed_away - device - {} - pending - armed_away - None".format(
-        "alarm_control_panel.entity"
+    assert (
+        calls[3].data["some"]
+        == "armed_away - device - alarm_control_panel.entity - pending - armed_away - None"
     )
 
     # Fake that the entity is armed night.
     hass.states.async_set("alarm_control_panel.entity", STATE_ALARM_PENDING)
     hass.states.async_set("alarm_control_panel.entity", STATE_ALARM_ARMED_NIGHT)
     await hass.async_block_till_done()
     assert len(calls) == 5
-    assert calls[4].data[
-        "some"
-    ] == "armed_night - device - {} - pending - armed_night - None".format(
-        "alarm_control_panel.entity"
+    assert (
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5054188)